feat(icons): DLT 2494 update dialpad logo gradient

jcarter-dp opened the pull request:
# feat(icons): DLT 2494 update dialpad logo gradient and empty state illustration

## Obligatory GIF (super important!)

<!--- go to giphy.com, pick a gif, click share, copy gif link, paste within the () brackets below. -->

![Obligatory GIF](path/to/gif)

## :hammer_and_wrench: Type Of Change

- [x] Feature

## :book: Jira Ticket

<!--- Enter the URL of the Jira ticket associated with this PR -->

## :book: Description

<!--- Describe specifically what the changes are -->

## :bulb: Context

<!--- Describe the purpose of the changes -->
<!--- Why did we make these changes? -->
<!--- What problem(s) do they solve? -->

## :pencil: Checklist

<!--- Tick or place an `x` in all of the checkboxes that apply -->
<!--- Remove checkboxes that do not apply -->

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :crystal_ball: Next Steps

<!--- Describe any future changes that need to be made after merging the PR, especially any follow up tasks after release. -->

## :camera: Screenshots / GIFs

<!--- Add these if necessary. Since we have deploy previews for every PR it may not always be. -->
<!--- Link any screenshots / GIFs below -->

## :link: Sources

<!--- Add any links to external reference material -->

francisrupert commented:
While I've hardcoded the grid of components, this is ready for publication (pending your review). The HTML/CSS for that grid is solid, and will likely be reused for a landing page for each top-level page—at the very least Design and Guides.

I'll log a task with the task to do it up better; specifically, hook it up to pull from front-matter (presumably).

braddialpad commented on docs/assets/less/dialtone-docs.less:
```diff
@@ -406,6 +405,121 @@ nav a.router-link-active {
+      grid-template-columns: [full-start] repeat(2, [col-start] var(--col-width, minmax(0, 1fr)) [col-end]) [full-end] !important;
+    }
+
+    @media screen and (min-width: 768px) {
+      grid-template-columns: [full-start] repeat(3, [col-start] var(--col-width, minmax(0, 1fr)) [col-end]) [full-end] !important;
+    }
```
Would you say 768 is a pretty common breakpoint? Just wondering as we don't include it in our responsive breakpoints https://dialpad.design/utilities/responsive/breakpoints.html#classes

francisrupert replied:
Yeah, that was me being kinda picky about it. 768 is extremely common as a breaking point, e.g. iPad portrait width. Note I'm using media queries going up rather than down as we appear to be doing everywhere. 

I'm fine with it as a one-off here, though at some point (🙄) we should be very intentional about rwd approach, and we just might reach using using [container queries](https://caniuse.com/css-container-queries) before then :)

braddialpad reviewed (approved):
Looks great! Tested at all screen sizes.

jcarter-dp committed:
Update dialpad-logo.svg

Added icon with proper gradient in svg.

jcarter-dp committed:
Updated empty state with new illustration

Updated doc component page empty state. Replaced previous illiustration with new rebrand "nothing to see here" illustration.

jcarter-dp committed:
Update empty state png image

Image with a bit pixelated. Fixed it

francisrupert commented on apps/dialtone-documentation/docs/components/empty-state.md:
```diff
@@ -140,7 +140,7 @@ Appropriate for indicating that something is currently unavailable to them. Prov
 ## Variants
 
-A few rules to keep in mind when choosing a size variant.
+A few rules to keep in nothing-to-see-here when choosing a size variant.
```
Oops.

francisrupert commented on packages/dialtone-icons/src/svg/icons/brand-full-color/dialpad-logo.svg:
Perfect.

francisrupert reviewed (approved):
Approving, just one little copy fix: https://github.com/dialpad/dialtone/pull/729#pullrequestreview-2785522662
